src/services/FileActions.js

```diff
@@ -0,0 +1,105 @@
+/**
+ * @copyright Copyright (c) 2019 John Molakvoæ <skjnldsv@protonmail.com>
+ *
+ * @author John Molakvoæ <skjnldsv@protonmail.com>
+ *
+ * @license AGPL-3.0-or-later
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+import { encodePath } from '@nextcloud/paths'
+import { generateUrl } from '@nextcloud/router'
+import { showError } from '@nextcloud/dialogs'
+import axios from '@nextcloud/axios'
+import { getCurrentUser } from '@nextcloud/auth'
+
+import client from './DavClient.js'
+import logger from './logger.js'
+
+/**
+ * Delete a file
+ *
+ * @param {number} fileName - The file's id
+ */
+export async function deleteFile(fileName) {
+	try {
+		await client.deleteFile(`/files/${getCurrentUser()?.uid}/${fileName}`)
+	} catch (error) {
+		logger.error(t('photos', `Fail to delete ${fileName}.`), error)
+		showError(t('photos', `Fail to delete ${fileName}.`))
```

Comment:
```suggestion
		logger.error(t('photos', 'Failed to delete {fileName}.', { fileName }), error)
		showError(t('photos', 'Failed to delete {fileName}.', { fileName }))
```

Reply:
Not resolved?

Reply:
Ah ! Though it was Github rendering twice the same review :see_no_evil: Fixed now.